app/routes/projects/$projectId/index.tsx

```diff
@@ -644,6 +605,45 @@ export default function ProjectDetailsPage() {
         handleCloseModal={() => setShowJoinModal(false)}
       />
 
+      <Container sx={{ marginBottom: 2 }}>
```

Comment:
I think that we can do a new component based on this code in order to use it on other parts of the project

Reply:
Also if there is no applicants the containers looks like this 
<img width="1169" alt="image" src="https://github.com/wizeline/remix-project-lab/assets/74632242/29745c04-babe-40e4-9213-34a647d14391">
without a card component, if you move this to a new component we can improve that